test/new-e2e/pkg/environments/aws/kubernetes/eks.go

```diff
@@ -175,9 +175,14 @@ func EKSRunFunc(ctx *pulumi.Context, env *environments.Kubernetes, params *Provi
 			return err
 		}
 
+		if params.eksInitOnly {
```

Comment:
As you introduced the param `ddinfra:initOnly`, should we read from common env instead?
This can then set in a generic way when building default `ConfigMap` parameters and avoids checking `isInit` is many places.

Reply:
Updated to use value from common env when possible and to fill it when building the ConfigMap! Let me know if that does not match you meant